tests/ui/where-clauses/ignore-err-clauses.rs

```diff
@@ -4,7 +4,7 @@ fn dbl<T>(x: T) -> <T as Add>::Output
 where
     T: Copy + Add,
     UUU: Copy,
-    //~^ ERROR cannot find type `UUU` in this scope
+    //~^ ERROR cannot find type `UUU`
```

Comment:
Hmm, there's actually a lot of unnecessary error annotation changes like this that blow up the PR size.
Could you avoid all of them?